### ISSUE TITLE
指标值非常大？

### ISSUE THREAD

**Issue by WenHui-Zhou** [xueleichen/PSNR-SSIM-UCIQE-UIQM-Python#1]
你好，我用你的指标算了一下uiqm和uciqe，发现指标非常的大。据我看过的论文，这两指标通常在2以内。

这是我算后的结果：
Average: uiqm=9.089692147135265 uciqe=4.310251438111393

不知什么原因。

**Comment by xueleichen** [comment]
你好，感谢关注。
我也注意到了这一点，我是根据这两个指标的论文实现的，可能代码里有一些问题，我最近抽空再检查一下。

**Comment by leefly072** [comment]
您最有空吗可以更新代码吗！？谢谢

**Comment by xueleichen** [comment]
Hi,

I have updated the code for UIQM. Now UIQM values are much smaller and close to one. The major differences between the current version and the old version are: (1) the way to deal with zero division in EME calculation (2) logAMEE calculation

As for UCIQE, I checked my code and the paper again and could not find possible mistakes. Meanwhile, I also found a GitHub repo for [UIQCE](https://github.com/Owen718/Image-quality-measure-method/blob/main/UCIQE.m). The values reported by this repo are smaller than one. However, the La*b* values used by this repo are divided by 255. And the saturation calculation is a little different from mine.
